src/state/actions/auth.ts

```diff
@@ -0,0 +1,27 @@
+import railsAxios from '../../util/railsAxios';
+import { initialState } from '../index';
+
+export const logIn = async (store, {email, password}) => {
+  const {loginUrl, userIdentity} = store.state;
+
+  try {
+    const response = await railsAxios().post(
```

Comment:
this is probably the better way to format a request. I might even update the `API.js` file to be axios and async/await based like it is here.